bin/out

```diff
@@ -44,6 +44,39 @@ start_job() {
     # TODO make sure job is started
 }
 
+delete_resource() {
+    RESOURCE_TYPE=$1
+    RESOURCE=$2
+
+    [ -n "$RESOURCE_TYPE" ] || exit 1
+    [ -n "$RESOURCE" ] || exit 1
```

Comment:
it's basically : error you if you find an unassigned var

http://redsymbol.net/articles/unofficial-bash-strict-mode/
do read it ⬆️ , i sent it to you before, it's saved me a lot of time